autoload/SpaceVim/layers/checkers.vim

```diff
@@ -8,7 +8,7 @@ let s:STRING = SpaceVim#api#import('data#string')
 function! SpaceVim#layers#checkers#plugins() abort
   let plugins = []
 
-  if g:spacevim_enable_neomake
+  if g:spacevim_enable_neomake && g:spacevim_enable_ale == 0
     call add(plugins, ['neomake/neomake', {'merged' : 0, 'loadconf' : 1 , 'loadconf_before' : 1}])
   elseif g:spacevim_enable_ale
```

Comment:
I can imagine allowing to enable both Neomake and ALE (even though ALE might complain by default).
I am lazy-loading ALE sometimes after Neomake, just to test / see how it works by now.
I think having one of them doing autolinting, but calling the other manually (e.g. automaking with Neomake, but using ALE's fixer) makes sense.

Reply:
I still prefer use one of them by defaule. users can use custom plugins to add ale and ale's config.
but in spacevim, if neomake is enabled, ale will be disabled, and this config will not be applyed to ale. but users still can use spacevim's custom plugin option to add ale.

SpaceVim contains too many plugins now. even though most of them are lazzy loaded. but I user do not want to see plugins with same function are installed at same time.